test/behaviour/connection/ConnectionSteps.java

```diff
@@ -74,14 +77,23 @@ public synchronized void before() {
         String address = GraknSingleton.getGraknRunner().address();
         assertNotNull(address);
         client = new GraknClient(address);
+        client.databases().all().forEach(database -> client.databases().delete(database));
```

Comment:
It was irritating for local development, when a database already existed prior to running the tests - the only option was to open a console or similar and delete it, just to test the tests.

Now, we ensure a clean slate prior to running tests.

Reply:
yeah i think we do this in grakn core by deleting the server directory and creating a new one, looks fine to me